user_manual/android/android_app.rst

```diff
@@ -0,0 +1,115 @@
+button. (Click the eyeball to the right of your password to expose your 
+password.)
+
+.. figure:: /images/android-new-account.png
+
+For best security your ownCloud server should be SSL-enabled, so that you can 
+connect via ``https``. The ownCloud app will test your connection as soon as 
+you enter it and tell you if you entered it correctly. If your server has a 
+self-signed SSL certificate you'll get a scary warning how it is not to be 
+trusted. Click the OK button to accept the certificate and complete your account 
+setup.
+
+.. figure:: /images/android-ssl-cert.png
+
+Managing Files
+--------------
+
+Now you should see the Files page of your ownCloud account. Click the overflow 
+button at the top right (that's the one with three vertical dots, and that is 
+really what it is called) to open a user menu. ``Refresh account`` refreshes the 
+page view. ``Settings`` take you to your settings menu. ``Sort`` gives you the 
+option to sort your files by date, or alphabetically.
+
+.. figure:: /images/android-files-page.png
+
+The little file folder icon to the left of the overflow button opens a dialog to 
+create a new folder. The arrow button opens a file upload dialog, and you can 
+either upload content from other Android apps such as Google Drive, the Gallery, 
+your music player, or from your Android filesystem. When you add a new file 
+you will see a confirmation on the top left when it has uploaded successfully, 
+and it is immediately synchronized with the server.
+
+.. figure:: /images/android-upload.png
+
+All files (that you have permission to access) on your ownCloud server are 
+displayed in your Android app, but are not downloaded until you download them. 
+Downloaded files are marked with a green arrow.
+
+.. figure:: /images/android-file-list.png
+
+Download and preview a file with a short press on the filename.  When the file 
+is in preview mode, a short press on the overflow button opens a menu with 
+options for sharing, opening with an app, removing, sending, and displaying file 
+details. 
+
+.. figure:: /images/android-file.png
+
+
+A long press on the filename does not download it, but opens a dialog with 
+options for sharing, downloading, renaming, moving, removing, sending, and 
+viewing file details. 
+
+
+.. figure:: /images/android-file-options.png
+
+
+Settings
+--------
+
+The Settings screen offers a number of useful options. In the Accounts 
+section you can configure multiple ownCloud accounts.
+
+The Security section sets up strong two-factor authentication by allowing you 
+to add a PIN (personal identification number) to access your account.  
+
+The Instant Uploads section creates a directory, :file:`/InstantUpload`, and 
+any photos or videos created with your Android device's camera are instantly 
+uploaded to this directory. You also have the option to choose any other 
+existing directory. Another nice option is Upload Pictures/Video via WiFi Only, 
+to save precious cell minutes.
```

Comment:
Maybe instead of saying "save precious cell minutes" is more clear to say something like "save mobile Internet data usage"